utils/docker/build.sh

```diff
@@ -37,4 +37,8 @@ else
   exit 1
 fi
 
-docker build --platform "${PLATFORM}" -t "$3" -f "Dockerfile.$2" .
+PW_VERSION=$(node ../../utils/workspace.js --get-version)
```

Comment:
we do the same also here, lets replace it also with --get-version? https://github.com/microsoft/playwright/blob/76c37f1977bfdb31165e09db61b4c43d95a9ac7c/utils/docker/publish_docker.sh#L10